### PR TITLE
Use pyproject.toml

### PR DESCRIPTION
**Pull request by sjperkins**
Based on and supercedes:

- #64 

- [x] Tests added / passed

  ```bash
  $ py.test --flake8 -v -s .
  ```

  If the flake8 tests fail, the quickest way to correct
  this is to run `autopep8` and then `flake8`
  to fix the remaining issues.

  ```
  $ pip install -U autopep8 flake8
  $ autopep8 -r -i .
  $ flake8 .
  ```

### CODE REVIEW

**Review by o-smirnov — APPROVED** [review]
LGTM!

File: tests/test_regions.py [hunk 1]
```diff
@@ -7,14 +7,13 @@
 def test_regions(ds9_region_file):
     regions = load_regions(ds9_region_file)
 
-    c1 = SkyCoord(299.8452923, 40.7403572, frame='fk5', unit='deg')
-    assert regions[0].center.position_angle(c1) == Angle(0, unit='rad')
+    assert len(regions) == 3
```
Comment (sjperkins) [inline tests/test_regions.py:10]: @o-smirnov Can I get a quick eye-ball on the test case here? I just removed the `position_angle` test -- something has probably changed in the upstream astropy calculation but the actual SkyCoord value matches. We don't use `position_angle` in the code anywhere so I don't think this is a critical failure, more something

**Comment by sjperkins** [comment]
Oh it got merged :-D

File: tests/test_regions.py (hunk 1 above)
Comment (sjperkins) [reply tests/test_regions.py:10]: I used `position_angle` as the comparison criteria for whatever reason in the depths of time.

Comment (o-smirnov) [reply tests/test_regions.py:10]: Nah, it was always a misguided test. PA is completely undefined when two positions match -- it just happened to return 0 but is allowed to return any value. The new tests are valid.

**Pull request merged**